docs/developer_guide.md

```diff
@@ -173,7 +179,7 @@ export IMAGE="openvino/model_server:latest"
 - Example:
 
 ```bash
-os.environ["IMAGE"] = "openvino/model_server"
+echo 'os.environ["IMAGE"] = "openvino/model_server"' >> user_config.py
```

Comment:
Will this work out of the box? Don't we need:
`import os` ?

Reply:
I removed echo command and went back to previous version, since user_config.py is not used anywhere in file.